tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/IR/hlo_ops.cc

```diff
@@ -2348,6 +2696,71 @@ static LogicalResult Verify(DynamicPadOp op) {
   return success();
 }
 
+LogicalResult DynamicPadOp::inferReturnTypeComponents(
+    MLIRContext*, llvm::Optional<mlir::Location>, ValueRange, DictionaryAttr,
+    RegionRange, llvm::SmallVectorImpl<mlir::ShapedTypeComponents>&) {
+  return failure();
+}
+
+LogicalResult DynamicPadOp::reifyReturnTypeShapes(
```

Comment:
Where is this function (`reifyReturnTypeShapes`) tested?

Reply:
This filecheck-based ut will test this:
https://github.com/tensorflow/tensorflow/pull/49598/files#diff-a6fc1c1e273cf5234d691f9a691f08b8cfbc9c07107558dda12a2009606d7fd6R56